setup.py

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env python3
 
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
```

Comment:
I don't mind, but out of curiosity, is this automatic reordering from a tool like PyCharm or something?